src/main/java/com/cinema/dao/OrderDao.java

```diff
@@ -0,0 +1,11 @@
+package com.cinema.dao;
+
+import com.cinema.model.Order;
+import com.cinema.model.User;
+import java.util.List;
+
+public interface OrderDao {
+    Order create(Order order);
+
+    List<Order> getOrderHistory(User user);
```

Comment:
I suppose in this case we better use a more certain name of the method, getOrdersOfUser or getUsersOrders, etc.